tools/cldr-apps/src/test/java/org/unicode/cldr/unittest/web/TestXPathTable.java

```diff
@@ -133,13 +133,15 @@ public void TestNonDistinguishing() throws SQLException {
             "numbers=hebrew",
             "//ldml/dates/calendars/calendar[@type=\"chinese\"]/dateFormats/dateFormatLength[@type=\"medium\"]/dateFormat[@type=\"standard\"]/pattern[@type=\"standard\"][@alt=\"variant-proposedx333\"][@numbers=\"hanidec\"][@draft=\"true\"]",
             "numbers=hanidec",
+            "//ldml/personNames/foreignSpaceReplacement[@xml:space=\"preserve\"]",
+            "xml:space=preserve",
```

Comment:
i thought the goal was to NOT have this listed in the results from `getSpecialNondistinguishingAttributes()`

Reply:
Hmm, you may be right. I was assuming that non-distinguishing attributes should *not* be shown. But I see that the numbers=hanidec attribute shown above is also displayed in blue in the ST: https://st.unicode.org/cldr-apps/v#/zh/Chinese/4c684156dd09b01c

Reply:
So maybe I *do* need to add xml:space to the explicit list in this function, which are things that should be treated as distinguishing even though that are not actually distinguishing. Seems backward but maybe I do not really understand the intent of the function.